index.html

```diff
@@ -0,0 +1,155 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <link rel="stylesheet" href="styles/styles.css">
+    <meta name="viewport" content="width=device-width, initial-scale=0.5">
+    <title>Les 4</title>
+</head>
+<body>
+<header class="menu outer-container">
+    <nav>
+        <div class="inner-container-wide">
+            <ul>
+                <li><a href="">Case Studies</a></li>
+                <li><a href="">Services</a></li>
+                <li><a href="">Contact us</a></li>
+            </ul>
+        </div>
+    </nav>
+
+    <div class="banner inner-container-narrow">
+
+        <h1>
+            Your website is an investment in the success of your business.
+        </h1>
+
+        <p>
+            We are here to help you succeed.
+        </p>
+
+        <a href="" class="button">
+            Let's work together
+        </a>
+    </div>
+
+    <div class="squiggle-container">
+        <img class="squiggle" src="assets/squiggle_line_stripe.svg" width="50px" height="50px"/>
+    </div>
+
+</header>
+<main class="outer-container">
+    <section>
+        <div class="inner-container-narrow">
+            <p class="about-us">
+                Rekupe is a digital agency in Los Angeles.
+                <br>
```

Comment:
Self-closing tag gebruiken
```suggestion
                <br />
```